pkg/virt-handler/migration-proxy/migration-proxy.go

```diff
@@ -92,6 +94,16 @@ func SourceUnixFile(virtShareDir string, key string) string {
 	return filepath.Join(virtShareDir, "migrationproxy", key+"-source.sock")
 }
 
+func ipAnyAddress() string {
```

Comment:
see question below on the function like this please

Reply:
saw that Alona updated is ipv6 enable func, and saw there are some remarks to save state and such,
once it will be done i will align the 2 uses of this func